tools/PI/DevHome.PI/Pages/WERPage.xaml

```diff
@@ -0,0 +1,96 @@
+                    <Button Visibility="{x:Bind ViewModel.AllowElevationOption}" Command="{x:Bind ViewModel.RunAsAdminCommand}">
+                        <StackPanel Orientation="Horizontal" Spacing="10">
+                            <SymbolIcon Symbol="Admin"/>
+                            <TextBlock x:Uid="RunElevatedButtonForWER" Margin="0"/>
+                        </StackPanel>
+                    </Button>
+                    <HyperlinkButton x:Uid="LocalDumpMoreInfo" NavigateUri="https://learn.microsoft.com/windows/win32/wer/collecting-user-mode-dumps" />
+                </StackPanel>
+            </Grid>
+        </StackPanel>
+
+
+        <controls:DataGrid
+            x:Name="WERDataGrid" Grid.Row="1" Margin="0,6,0,0"
+            VerticalScrollBarVisibility="Visible"
+            AlternatingRowBackground="Transparent"
+            AlternatingRowForeground="Gray"
+            AreRowDetailsFrozen="False"
+            AreRowGroupHeadersFrozen="True"
+            AutoGenerateColumns="False"
+            CanUserSortColumns="True"
+            CanUserReorderColumns="True"
+            CanUserResizeColumns="True"
+            ColumnHeaderHeight="32"
+            FrozenColumnCount="0"
+            GridLinesVisibility="None"
+            HeadersVisibility="Column"
+            SelectionChanged="WERDataGrid_SelectionChanged"
+            Sorting="WERDataGrid_Sorting"
+            ItemsSource="{x:Bind ViewModel.DisplayedReports, Mode=OneWay}">
+            <controls:DataGrid.Columns>
+                <controls:DataGridTextColumn Tag="DateTime" x:Uid="WERTimeGeneratedColumn" Binding="{Binding Report.TimeGenerated}" Width="180"/>
+                <controls:DataGridTextColumn Tag="FaultingExecutable" x:Uid="WERFaultingExecutableColumn" Binding="{Binding Report.Executable}" Width="180"/>
+                <controls:DataGridTextColumn Tag="WERBucket" x:Uid="WERBucketColumn" Binding="{Binding FailureBucket, Mode=OneWay}" Width="240"/>
+                <controls:DataGridTemplateColumn Tag="CrashDumpPath" x:Uid="WERDumpColumn" Width="*">
+                    <controls:DataGridTemplateColumn.CellTemplate>
+                        <DataTemplate>
+                            <HyperlinkButton Content="{Binding Report.CrashDumpPath, Mode=OneWay}" Click="HyperlinkButton_Click" Tag="{Binding}"/>
+                        </DataTemplate>
+                    </controls:DataGridTemplateColumn.CellTemplate>
+                </controls:DataGridTemplateColumn>
+            </controls:DataGrid.Columns>
+        </controls:DataGrid>
+
+        <StackPanel Grid.Row="2" Orientation="Vertical">
+            <SelectorBar x:Name="InfoSelector" SelectionChanged="SelectorBar_SelectionChanged">
+                <SelectorBarItem x:Uid="DumpInfoSelectorItem" IsSelected="True"/>
+            </SelectorBar>
+            <ScrollViewer Margin="0,10,0,0" VerticalScrollBarVisibility="Auto">
```

Comment:
Suggest Spacing on the StackPanel instead of Margin here.

Reply:
Thanks!